integration-cli/docker_cli_network_unix_test.go

```diff
@@ -768,6 +768,30 @@ func (s *DockerNetworkSuite) TestDockerNetworkDriverOptions(c *check.C) {
 
 }
 
+func (s *DockerNetworkSuite) TestDockerPluginV2NetworkDriver(c *check.C) {
+	testRequires(c, DaemonIsLinux, ExperimentalDaemon, Network)
+
+	var (
+		npName        = "mavenugo/test-docker-netplugin"
+		npTag         = "latest"
+		npNameWithTag = npName + ":" + npTag
+	)
+	_, _, err := dockerCmdWithError("plugin", "install", "--grant-all-permissions", npNameWithTag)
```

Comment:
atm, DockerSuite's teardowntests's dont cleanup plugins. So please handle `plugin rm` in a defer.